examples/counter/tauri/src/App.tsx

```diff
@@ -0,0 +1,65 @@
+import { useEffect, useState } from "react";
+import { invoke } from "@tauri-apps/api"
+import { listen, UnlistenFn } from "@tauri-apps/api/event"
+import "./App.css";
+
+type State = {
+  text: string;
+};
+
+const initialState: State = {
+  text: "",
```

Comment:
Technically, this is missing the `confirmed` field, which allows the UI to style optimistic updates differently (e.g. iOS makes the text gray). Not a big deal though, not all of the examples use that field.

Reply:
This is a funny one. The confirmed field does come in with updates, regardless of this type, so even though it's not here, this version still does the "pending" thing. But I agree it should be included for clarity.

Reply:
Funny, looks like I copied this initial state type/value from the `web-nextjs`. Should we update that example as well?I think it makes sense to keep them as similar as possible for maintenance and learning.

See https://github.com/redbadger/crux/blob/master/examples/counter/web-nextjs/pages/index.tsx#L27

Reply:
I haven't updated this type for now, for consistency with the other example, but it's an easy change in both places if we want to make it.

Reply:
Yea I noticed the web example has the same issue. Let's keep this as is for now, we can always fix it later on.